.env

```diff
@@ -1,4 +1,4 @@
-REACT_APP_API_URL=/
-REACT_APP_AZURE_AD_CLIENT=abc3be17-5b4c-48c7-9cd8-db28399d772b
-REACT_APP_AZURE_AD_TENANT=0251ecdb-4e2d-4077-9498-40328ac794e9
-REACT_APP_FONT_URL=https://cloud.typography.com/7763712/615144/css/fonts.css
\ No newline at end of file
+VITE_API_URL=/
+VITE_AZURE_AD_CLIENT=abc3be17-5b4c-48c7-9cd8-db28399d772b
+VITE_AZURE_AD_TENANT=0251ecdb-4e2d-4077-9498-40328ac794e9
+VITE_FONT_URL=https://cloud.typography.com/7763712/615144/css/fonts.css
```

Comment:
Do these three really need to start with VITE (or REACT)? Can't they just be AZURE_AD_CLIENT, AZURE_AD_TENANT, etc?

Reply:
Yes, they do. CRA required that any variables available to the client start with `REACT_`, whereas Vite requires that they start with `VITE_`.

[According to Vite](https://vitejs.dev/guide/env-and-mode.html#env-fileshttps://vitejs.dev/guide/env-and-mode.html#env-files), this is a security mechanism:

> To prevent accidentally leaking env variables to the client, only variables prefixed with VITE_ are exposed to your Vite-processed code.

Reply:
Thanks for clarifying!